sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOStorageReadTest.java

```diff
@@ -1368,19 +1368,14 @@ public void testStreamSourceSplitAtFractionFailsWhenParentIsPastSplitPoint() thr
   public void testReadFromBigQueryIO() throws Exception {
     fakeDatasetService.createDataset("foo.com:project", "dataset", "", "", null);
     TableReference tableRef = BigQueryHelpers.parseTableSpec("foo.com:project:dataset.table");
-
-    Table table =
-        new Table().setTableReference(tableRef).setNumBytes(10L).setSchema(new TableSchema());
-
+    Table table = new Table().setTableReference(tableRef).setNumBytes(10L).setSchema(TABLE_SCHEMA);
     fakeDatasetService.createTable(table);
 
     CreateReadSessionRequest expectedCreateReadSessionRequest =
         CreateReadSessionRequest.newBuilder()
             .setParent("projects/project-id")
             .setTableReference(BigQueryHelpers.toTableRefProto(tableRef))
             .setRequestedStreams(10)
-            .setReadOptions(
```

Comment:
Is this a feature regression for the storage API based read path ?

Reply:
I'm not sure that I understand your question here. We used to have e2e tests for BigQueryIO with the read API only for the case where the caller specified all fields as selected fields; with this change, now we have two tests -- one which covers the case where no selected fields are covered (which is effectively the same as the case where all fields are specified), and another which covers the case where only a subset of fields are specified.